finetune/base.py

```diff
@@ -194,56 +101,89 @@ def _target_model(self, *, featurizer_state, targets, n_outputs, train=False, re
         # Overridden by subclass to attach a target model onto the shared base featurizer.
         raise NotImplementedError
 
-    @abstractmethod
-    def _target_encoder(self):
-        # Overridden by subclass to produce the right target encoding for a given target model.
-        raise NotImplementedError
-
-    def _eval(self, *tensors, feed_dict):
-        """
-        Evaluate the value of each of the provided tensors.
-        Returns a `dict` that maps from tensor to result value.  
-        If any result value is None, that result is excluded from the results `dict`.
-        """
-        tensors = [
-            tensor if tensor is not None else self.noop
-            for tensor in tensors
-        ]
-        values = self.sess.run(tensors, feed_dict=feed_dict)
-        return {
-            tensor: value
-            for tensor, value in zip(tensors, values)
-            if value is not None
-        }
-
     def finetune(self, Xs, Y=None, batch_size=None):
```

Comment:
I think we removed batch_size at some point in favor of always opting to use config.batch_size

Reply:
https://github.com/IndicoDataSolutions/finetune/blob/92b28ecc1db0e33838995ea792b5d3b6f3dd7686/finetune/base.py#L219

Reply:
Hrmmmm -- k.  I was probably thinking of max_length.